java/ql/src/semmle/code/java/security/OgnlInjection.qll

```diff
@@ -0,0 +1,130 @@
+/** Provides classes to reason about OGNL injection vulnerabilities. */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.ExternalFlow
```

Comment:
There also needs to be a bidirectional import in `ExternalFlow.qll`

Reply:
I thought this was enough: https://github.com/github/codeql/pull/5894/files#diff-f87085fa948d6f4323ebfe0492af2d1f2b1bcefe51cf24088789a6f44b829478R84

Is anything else needed?

Reply:
In the file `ExternalFlow.qll` file there's a private module named `Frameworks` that imports every file containing CSV rows. This file (`semmle.code.security.OgnlInjection`) should be added there too.

Reply:
Yes; never mind; I missed that previously.